.github/workflows/workflow_path_to_live.yml

```diff
@@ -110,13 +110,23 @@ jobs:
       version_tag: ${{ needs.create_tags.outputs.version_tag }}
     secrets: inherit
 
+  production_smoke_tests:
```

Comment:
I've been thinking about this, maybe we should just call them "production tests" (so under `cypress/production`), since that is what they are for?

Reply:
Yeah was chatting this through with @andrewpearce-digital earlier - we're going to add in another check before preproduction cypress tests run to catch any misconfigurations before we deploy to prod so would agree the name change makes sense.

I'll still leave the prod test as well in case the tf config for preproduction drifts out of sync over time.